front/src/applications/stdcmV2/components/StdcmEmptyConfigError.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { type FC } from 'react';
+
+import { Tools } from '@osrd-project/ui-icons';
+import { useTranslation } from 'react-i18next';
+
+const StdcmEmptyConfigError: FC<unknown> = () => {
```

Comment:
Outside of the editor, we don't use FC. Use a simple const to be consistent with the rest of the code base
```suggestion
const StdcmEmptyConfigError = () => {
```